packages/cli/src/config.rs

```diff
@@ -77,6 +77,12 @@ impl Default for DioxusConfig {
                     style: Some(vec![]),
                     script: Some(vec![]),
                 },
+                https: WebHttpsConfig {
```

Comment:
It looks like this config is unused, and `https://localhost:8080` does not appear to work locally with this branch.

Maybe you forgot to push some changes or some changes that used this got overriten?

Reply:
I hate to say this but, are you sure it's not something on your end?

Config is used:
https://github.com/DioxusLabs/dioxus/blob/d83b00ff0f4a51e27e4fab33fd1c6d25a7caac0f/packages/cli/src/server/mod.rs#L214

Localhost HTTPs: 
![image](https://github.com/DioxusLabs/dioxus/assets/30190859/501a6d42-01c8-4ee0-a86e-b99095037a06)

No missing commits on my end.

Reply:
Sorry, I was searching the git diff, but one file was collapsed

Reply:
If you need it, this cargo install command works targeting my branch.
```
cargo install --debug --git https://github.com/DogeDark/dioxus.git --branch cli-https dioxus-cli
```